.changeset/tender-windows-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"wb-dev-build-settings": major
```

Comment:
**note:** I'm not fully sure if this should be marked as a `major`. It's a breaking change for this package, but it's not a "public" package, so I don't know how to bump this, and how that will affect the WB packages that depend on this (as a dev dep).